apps/root/src/frame/polling-handlers/earn-positions-updater/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import useInterval from '@hooks/useInterval';
+import useEarnService from '@hooks/earn/useEarnService';
+import { IntervalSetActions } from '@constants/timing';
+
+const EarnPositionsUpdater = () => {
+  const earnService = useEarnService();
+  const updateInterval = IntervalSetActions.earnPositionsUpdate;
+
+  const updateBalancesAndPrices = React.useCallback(async () => {
+    await earnService.fetchUserStrategies();
```

Comment:
Be carefull since this should extend the positions that we have right now, so for example it would remove the history from the detailed userStrategies